tests/validate/04-builtin-suites.t

```diff
@@ -39,9 +39,17 @@ with open(file_name, 'r') as in_file:
 set_test_number $((( ((${#SUITES[@]})) * 2 )))
 #-------------------------------------------------------------------------------
 # Validate suites.
+NO_EMPY=true
+if cylc check-software 2>'/dev/null' | grep -q '^Python:EmPy.*([^-]*)$'; then
+    NO_EMPY=false
+fi
 for suite in ${SUITES[@]}; do
     suite_name=$(sed 's/\//-/g' <<<"${suite:$ABS_PATH_LENGTH}")
     TEST_NAME="${TEST_NAME_BASE}${suite_name}"
+    if "${NO_EMPY}" && [[ "$(head -1 "${suite}")" == '#!EmPy' ]]; then
```

Comment:
You probably want regex matching like [this one](https://github.com/cylc/cylc/blob/master/lib/parsec/fileparse.py#L267):

```
head -1 "${suite}" | grep -q '^#![Ee]m[Pp]y'
```

Reply:
Fair enough. Updated with `grep -qi '^#!empy' < <(head -1 "${suite}")`.